ferretdb/ferretdb.go

```diff
@@ -80,13 +86,6 @@ func New(config *Config) (*FerretDB, error) {
 	version.Get().Package = "embedded"
 
 	sp, err := state.NewProviderDir(config.StateDir)
-	if err == nil {
-		// TODO https://github.com/FerretDB/FerretDB/issues/4750
-		err = sp.Update(func(s *state.State) {
-			s.TelemetryLocked = true
-		})
```

Comment:
That's exactly the part of the code that should be updated, not removed

Reply:
I have a minor question here. I have reviewed the code and noticed that most fields in 'state' are filled during `NewReporter()` and `makeReport()`, but `DocumentDBVersion` and `PostgreSQLVersion` are left empty. I haven't been able to find a method to obtain these two values in embedded FerretDB instance. Could you please advise if there are any other members in 'state' that need updating during the init embedded instace `Update` func? :)

Reply:
@AlekSi are there other fields of `State` that should be updated? Do we want to set `s.TelemetryLocked = true` only when `EnabledTelemetry` is unset?